File: Easy/Java/1475. Final Prices With a Special Discount in a Shop.java
class Solution {
    public int[] finalPrices(int[] prices) {
        for(int i = 0; i < prices.length; i++){
            if(i + 1 == prices.length)
                break;
            for(int j = i + 1; j < prices.length; j++){
                if(prices[j] <= prices[i]){
                    prices[i] -= prices[j];
                    break;
                }
            }
        }

        return prices;
    }
}

File: Easy/Python/1475. Final Prices With a Special Discount in a Shop.py
class Solution(object):
    def finalPrices(self, prices):
        for i in range(len(prices)):
            if i + 1 == len(prices):
                break
            for j in range(i + 1, len(prices)):
                if prices[j] <= prices[i]:
                    prices[i] -= prices[j]
                    break
        return prices
